tap_typeform/streams.py

```diff
@@ -240,11 +245,31 @@ def sync_latest_forms(atx):
     return state
 
 
+def is_incremental(stream):
+    return schemas.REPLICATION_METHODS[stream].get("replication_method") == "INCREMENTAL"
+
+
+def get_replication_key(stream):
+    if is_incremental(stream):
+        return schemas.REPLICATION_METHODS[stream].get("replication_keys")[0]
+    return None
+
+
+def construct_bookmark(stream_name, value):
+    replication_key = get_replication_key(stream_name)
+    if replication_key:
+        bookmark_value = {replication_key: value.to_datetime_string()}
+    else:
+        bookmark_value = value
+
+    return bookmark_value
+
+
 def sync_forms(atx):
     incremental_range = atx.config.get('incremental_range')
 
     for form_id in atx.config.get('forms').split(','):
-        bookmark = atx.state.get('bookmarks', {}).get(form_id, {})
+        # bookmark = atx.state.get('bookmarks', {}) atx.stream_name, {})
```

Comment:
Could you expand on why this is no longer needed?

Reply:
I had to change how the tap was bookmarking streams and included the form_id as a key. So for example, instead of the previous way which looked like this:
```
{'answers': 'some-datetime'}
```
It now looks like this:
```
{'answers': {'form_id_1': 'some-datetime'}, ... }
```
And so I also needed to change how the bookmark was retrieved. I mean to delete that line, not comment it out. I am using the `singer.get_bookmark` method on line 314 so this bookmark variable on 272 is no longer needed.